src/main/resources/database/changes/release-10.49.16/add_job_completed_column.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE sample.collectionexercisejob ADD COLUMN jobcomplete boolean;
```

Comment:
Do collectionExerciseJobs currently in the db need updating to false or true?

Reply:
have updated script to set all pre-existing jobs to jobcomplete = true